packages/block-editor/src/components/image-editor/index.js

```diff
@@ -26,14 +26,7 @@ export default function ImageEditor( {
 	borderProps,
 } ) {
 	return (
-		<ImageEditingProvider
-			id={ id }
-			url={ url }
-			naturalWidth={ naturalWidth }
-			naturalHeight={ naturalHeight }
-			onSaveImage={ onSaveImage }
-			onFinishEditing={ onFinishEditing }
-		>
+		<ImageCropperProvider>
```

Comment:
`IsEditSaveInProgress` needs to be somewhere in the state tree.

Reply:
Or, we wrap the current provider in the new  image cropper provider for now instead of trying to harmonize everything